CONTRIBUTING.md

```diff
@@ -156,6 +156,28 @@ Suppose your plugin is ready for version 1.2.0
 
 ```
 
+### Test Development
+
+#### Experimental Features
+
+When writing tests for experimental features, please follow these steps.
+
+1. figure out the folder location to put the tests
+
+Similar to the regular tests, OSD Core tests go to the [folder](integration/core-opensearch-dashboards/opensearch-dashboards/) and OSD plugin tests go to the [folder](cypress/integration/plugins/).
+
+2. develop tests with a flag to turn on and off
+
+Add an environment variable (e.g boolean) to only run tests for the experiemental feature when it is true. This is to ensure backward compatibility when integrating with [opensearch-build repo](https://github.com/opensearch-project/opensearch-build/blob/main/src/test_workflow/integ_test/service_opensearch_dashboards.py) whose OpenSearch Dashboards execution command or yml configuration may not be updated to support the experimental feature yet.
+
+3. set up Github action to run the tests inside the current repo
+
+Create a new workflow by referring to [this template](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/.github/workflows/release-e2e-workflow-template.yml) for OSD plugin or [this workflow](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/.github/workflows/cypress-workflow-vanilla-snapshot-based.yml) for OSD Core. This workflow is to run the OSD from artifact. You could enable your experimental feature through either `./bin/opensearch-dashboards` or through modifying the content of the yml file. (In order to run the tests from the source code repo of the feature, you can set up workflows to check out the source code and use `yarn` to start OSD.)
+
```

Comment:
Is it possible to link some example of how to "enable your experimental feature through `./bin/opensearch-dashboards` "?

Reply:
Is this a generic question for running OSD experimental feature or specifically related to tests?

Reply:
I think @kristenTian is asking if you can add a complete example command like 

```
./bin/opensearch-dashboards --wizard.enabled
```

Since its not always easy to remember how pass config values from project to project. in OS Engine its passed using the `-e` flag.

Reply:
Sure. updated with a complete example.